integration_test/cases/client_test.exs

```diff
@@ -104,7 +102,10 @@ defmodule ClientTest do
 
     assert P.run(pool, fn(conn) ->
       assert_receive :reconnected
-      assert {:error, %RuntimeError{}} = P.execute(conn, %Q{}, [:param])
+
+      assert {:error, %DBConnection.ConnectionError{message: "connection is closed"}} =
+               P.execute(conn, %Q{}, [:first])
```

Comment:
Because the ets entry is removed, this command will never work, so we never get to invoke the agent. So I changed the return result here and removed the :disconnect entry from the agent.

Reply:
I would say this whole test is useless, because the scenario "when client timeout and then disconnects" is no longer possible.